dist/seeThru.min.js

```diff
@@ -1,2 +1,2 @@
-/*! seethru 3.1.1 14-08-2017 see https://github.com/m90/seeThru for details */
```

Comment:
Can we undo these changes as well so that the diff covers `src/seeThru.js` only? Thanks.